oops/hosts/cassini/vims.py

```
@@ -189,7 +189,7 @@
 # Standard class methods
 ##########################################################################################
 
-def from_file(filespec, data=True):
+def from_file(filespec, method='strict', data=True):
```

Ruff (0.15.12)

[warning] Too many branches (52 > 12)

(PLR0912)